specification/eventgrid/data-plane/Microsoft.Media/stable/2018-01-01/MediaServices.json

```diff
@@ -924,7 +944,7 @@
         }
       },
       "type": "object",
-      "description": "Ingest fragment dropped event data. Schema of the data property of an EventGridEvent for a Microsoft.Media.LiveEventIngestHeartbeat event."
+      "description": "Ingest heartbeat event data. Schema of the data property of an EventGridEvent for a Microsoft.Media.LiveEventIngestHeartbeatEventData event."
```

Comment:
Need some clarity here - 
Did the event name change from `LiveEventIngestHeartbeat` to `LiveEventIngestHeartbeatEventData`? Or, is it just a documentation error?
cc: @Sanjidhalim

Reply:
Event name should not have changed. Seems like a documentation error. @xuepingd  is driving the PR right now. She can correct this